Lib/test/test_unicode.py

```diff
@@ -1355,7 +1390,7 @@ def test_formatting(self):
 
         self.assertEqual('%c' % 0x1234, '\u1234')
         self.assertEqual('%c' % 0x21483, '\U00021483')
-        self.assertRaises(OverflowError, "%c".__mod__, (0x110000,))
+        self.assertRaises(OverflowError, "%c".__mod__, (INVALID_CODEPOINT,))
```

Comment:
`FIRST_INVALID_CODEPOINT` would I think capture the intent more precisely. (This is similar to a remark @serhiy-storchaka made elsewhere.)